src/components/CardLog/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Column from '../Column'
+import Row from '../Row'
+import Image from '../Image'
+import './index.css'
+
+const CardLog = props => {
+  return (
+    <div className='CardLog__container'>
+      <Row>
+        {props.cards
+          .filter((_, index) => index < 6)
+          .map((i, index) => {
+            return (
+              <Column key={i.id + ' #' + index} width={25}>
```

Comment:
`25` only works for 4 columns. If you want 6, you’ll have to define something like `17`. Note that this is not a direct width applied to the column. It’s an indicator injected into a `Column--25` class name, which has associated styles. So if you’re going to add `17`, you also need to add a new class to `Column/index.css` like:

```css
.Column--17 { width: calc(100% / 6) }
```

I’m also happy reworking this `width` property to use fractions instead: `width='1/3`, which will yield classes like `Column--1/3` (perfectly valid CSS), so it’s a bit more explicit.

Reply:
That being said, I’m not even sure we should use rows and columns for a case like this. I think we might want to implement our own little layout within that container. I came up with something like this:

```jsx
const CardLog = props => (
  <div className='CardLog'>
    <ul className='CardLog__list'>
      {props.cards.map((i, index) => (
        <li key={index} className='CardLog__item'>
          <Image className='CardLog__image' src={i.image} alt={i.name} />
        </li>
      ))}
    </ul>
  </div>
)
```

```css
/**
 * 1. Fixed dimensions to make it easier to implement a scrollable layout. It
 *    should technically be possible to use full width, but it’s significantly
 *    harder because we’re in fluid containers at the moment (causing expanding
 *    and such).
 * 2. Inner spacing to act as a gutter between the items within, and the border.
 */
.CardLog {
  width: 350px; /* 1 */
  height: 55px; /* 1 */
  padding: 3px; /* 2 */
  background-color: var(--black);
  border: 1px solid var(--beige);
  border-radius: 6px;
}

/**
 * 1. Reset the default styles of an `ul` element.
 * 2. Take the entire height of the parent.
 * 3. Prevent content from wrapping, effectively forcing it on a single line.
 * 4. Enable horizontal scrolling with “snap points” for supported browsers, or
 *    regular horizontal scrolling otherwise.
 *    See: https://css-tricks.com/practical-css-scroll-snapping/
 */
.CardLog__list {
  padding: 0; /* 1 */
  margin: 0; /* 1 */
  list-style: none; /* 1 */
  height: 100%; /* 2 */
  white-space: nowrap; /* 3 */
  overflow-y: hidden; /* 3 */
  overflow-x: auto; /* 4 */
  scroll-snap-type: mandatory; /* 4 */
  scroll-snap-points-x: repeat(350px); /* 4 */
  scroll-snap-type: x mandatory; /* 4 */
}

/**
 * 1. Take the entire height of the parent.
 * 2. Treat items as a mix of inline and block elements, making it possible to
 *    line them all up on one single line.
 * 3. Slightly smaller border radius than the parent to align the inner and
 *    outer rounded corners nicely.
 *    See: https://joshnh.com/weblog/getting-inner-border-radius-just-right/
 * 4. Indicate that a snap should align with the left edge of an item.
 */
.CardLog__item {
  height: 100%; /* 1 */
  display: inline-block; /* 2 */
  border: 1px solid var(--dark-beige);
  border-radius: 4px; /* 3 */
  scroll-snap-align: start; /* 4 */
}

/**
 * 1. Add some spacing between items, equal to the `padding` of the top-level 
 *    container to have identical horizontal and vertical padding.
 */
.CardLog__item:not(:last-child) {
  margin-right: 3px; /* 1 */
}

.CardLog__image {
  display: block;
  height: 100%;
}

```